### PR TITLE
Re-answer Questionnaire

### PR DESCRIPTION
**Pull request by mnsison**
- added re-answer questionnaire page
->updates the Questionnaire table
->inserts new rows to PotentialMatch table

- modified css for Questionnaire page

- put each question and choices into a container
- made radio buttons bigger and to change color when hovering
-> users can click on the text as well to select the button
- changed the scroll bar colour

image
image

### CODE REVIEW

**Comment by jj7lin** [comment]
Instead of "Update Responses" it might be self explanatory to just change it to "Update". Looks a bit crowded in that small button.

Otherwise, i like it! Keep up the good work 👍

File: src/Questionnaire/UpdateQuestionnaire.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  render() {
+    if (this.state.id === "") {
+      this.props.history.push({
+        pathname: "/login",
+      });
+      return null;
+    }
+    return (
+      <div>
+          <Header id={this.state.userId} />
```
Comment (echanlee) [inline src/Questionnaire/UpdateQuestionnaire.js:104]: If you're using the same HTML/jsx for the `updateQuestionnaire.js` as the `Questionnaire.js`  then you can add the jsx into one file that can be rendered within both files to reduce redundant code

**Review by chalynpai — COMMENTED** [review]
- Could you change the scroll bar to the same colour as the update button?
- Increase box size of the questionnaire box

Good job! It looks good 👍

File: src/Questionnaire/UpdateQuestionnaire.js (hunk 1 above)
Comment (echanlee) [reply src/Questionnaire/UpdateQuestionnaire.js:104]: Not sure if that's possible ^ with our current state management but you can try

**Review by echanlee — APPROVED** [review]

**Pull request merged**